Gemfile.lock

```diff
@@ -1,10 +1,10 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    addressable (2.5.2)
+    addressable (2.6.0)
       public_suffix (>= 2.0.2, < 4.0)
     colorator (1.1.0)
-    ffi (1.9.25)
+    ffi (1.10.0)
     forwardable-extended (2.6.0)
     jekyll (3.4.3)
```

Comment:
Can you upgrade jekyll to 3.6.3 or higher, to get rid of the vulnerability?

Reply:
Sure! I was going to create a separate PR for that but can do it here

Reply:
_Bundler attempted to update jekyll but its version stayed the same_

ffs

Reply:
Done